tasks/db_deploy/test/test_db_deploy.py

```diff
@@ -239,33 +253,585 @@ def return_cursor_return_function(conn: connection):
+
+    @patch('db_deploy.execute_sql')
+    @patch('db_deploy.execute_sql_from_file')
+    @patch('db_deploy.get_cursor')
+    def test_create_schema_minimal_path(self,
+                                        mock_get_cursor: MagicMock,
+                                        mock_execute_sql_from_file: MagicMock,
+                                        mock_execute_sql: MagicMock):
+        mock_con = Mock()
+        mock_cursor = Mock()
+        mock_get_cursor.return_value = mock_cursor
+        os.environ[db_deploy.OS_ENVIRON_PLATFORM_KEY] = 'NOT AWS'
+
+        result = db_deploy.create_schema(mock_con)
+
+        mock_get_cursor.assert_called_once_with(mock_con)
+        mock_execute_sql.assert_not_called()
+        mock_execute_sql_from_file.assert_called_once_with(
+            mock_cursor, f"schema{db_deploy.SEP}app.sql", mock.ANY)
+        mock_con.commit.assert_called_once()
+        mock_cursor.close.assert_called_once()
+
+    @patch('db_deploy.execute_sql')
+    @patch('db_deploy.execute_sql_from_file')
+    @patch('db_deploy.get_cursor')
+    def test_create_schema_aws_auth(self,
+                                    mock_get_cursor: MagicMock,
+                                    mock_execute_sql_from_file: MagicMock,
+                                    mock_execute_sql: MagicMock):
+        """
+        Same as minimal path, but AWS requires extra session auth.
+        """
+        mock_con = Mock()
+        mock_cursor = Mock()
+        mock_get_cursor.return_value = mock_cursor
+        os.environ[db_deploy.OS_ENVIRON_PLATFORM_KEY] = 'AWS'
+
+        result = db_deploy.create_schema(mock_con)
+
+        mock_get_cursor.assert_called_once_with(mock_con)
+        mock_execute_sql.assert_called_once_with(mock_cursor, """SET SESSION AUTHORIZATION dbo;""", mock.ANY)
+        mock_execute_sql_from_file.assert_called_once_with(
+            mock_cursor, f"schema{db_deploy.SEP}app.sql", mock.ANY)
+        mock_con.commit.assert_called_once()
+        mock_cursor.close.assert_called_once()
+
+    @patch('db_deploy.execute_sql')
+    @patch('db_deploy.execute_sql_from_file')
+    @patch('db_deploy.get_cursor')
+    @patch('db_deploy.get_db_connection')
+    @patch('db_deploy.get_file_names_in_dir')
+    def test_create_tables_failed_script_does_not_halt(self,
+                                                       mock_get_file_names_in_dir: MagicMock,
+                                                       mock_get_db_connection: MagicMock,
+                                                       mock_get_cursor: MagicMock,
+                                                       mock_execute_sql_from_file: MagicMock,
+                                                       mock_execute_sql: MagicMock):
+        """
+        If one script fails, keep going.
+        """
+        db_host = uuid.uuid4().__str__()
+        db_name = uuid.uuid4().__str__()
+        db_port = uuid.uuid4().__str__()
```

Comment:
May want to use closer to valid data for port. port should be a number. For PostgreSQL default is 5432 but could use a random int converted to string.

Reply:
Makes sense. Fixed.